lib/portage/sync/syncbase.py

```diff
@@ -252,6 +252,12 @@ def _refresh_keys(self, openpgp_env):
 		"""
 		out = portage.output.EOutput(quiet=('--quiet' in self.options['emerge_config'].opts))
 
+		# Pass proxy settings in emerge configuration to Gemato ephemeral env
+		running_config_settings = self.options['emerge_config'].running_config.settings
+		for setting in ('http_proxy', 'https_proxy'):
```

Comment:
Does this grab values from environment if not set in make.conf too?

Reply:
Not at all. I preferred to keep explicit declaration of such parameters trough only the configuration file. Depending on the global portage policy, that could be adapted to use current env values, then if exists, the configuration ones.